hivemind/dht/node.py

```diff
@@ -38,6 +39,9 @@
 logger = get_logger(__name__)
 
 
+DHT_DEFAULT_NUM_WORKERS = os.environ.get("HIVEMIND_DHT_NUM_WORKERS", 8)
```

Comment:
```suggestion
DEFAULT_NUM_WORKERS = os.environ.get("HIVEMIND_DHT_NUM_WORKERS", 8)
```

We already know this is about `DHT` since it is located in the `dht` folder :)